### PR TITLE
https://argentinadev.atlassian.net/browse/RA-14

### PR DESCRIPTION
**Pull request by Solpio**

### CODE REVIEW

File: admin-app/src/constants/colors.ts [hunk 1]
```diff
@@ -39,6 +40,7 @@ export enum ColorsLight {
 	chatHoverBackgroundColor = "#C8C8C8",
 	inputCheckedColor = "#17A625",
 	inputUncheckedColor = "#BDBDBD",
+	breadcrumbsBorderColorActive = "#ff0000ff",
```
Comment (kurkul608) [inline admin-app/src/constants/colors.ts:44]: Are you sure the colors have to be the same?

File: admin-app/src/pages/chat-list-page/index.tsx [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
-      <ChatListWidget />
-    </>
-  );
+	const { t } = useTranslation("translation", { keyPrefix: "chatsPage" });
+	const location = useLocation();
+	return (
+		<>
+			<PageTitle>
+				<h3>{t("chats")}</h3>
+				<Breadcrumbs link={location.pathname}></Breadcrumbs>
```
Comment (kurkul608) [inline admin-app/src/pages/chat-list-page/index.tsx]: self-closing-comp

File: admin-app/src/pages/chat-settings/index.tsx [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
 export const ChatSettings = () => {
-  return <ChatSettingsWidget />;
+	const { t } = useTranslation("translation", { keyPrefix: "chatsPage" });
+	const location = useLocation();
+
+	return (
+		<>
+			<PageTitle>
+				<h3>{t("chats")}</h3>
+				<Breadcrumbs link={location.pathname}></Breadcrumbs>
```
Comment (kurkul608) [inline admin-app/src/pages/chat-settings/index.tsx]: self-closing-comp

File: admin-app/src/shared/components/breadcrumbs/services/link-parser.service.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+export function LinkParserService(link: string) {
+	let navTo = "";
+	console.log(link);
```
Comment (kurkul608) [inline admin-app/src/shared/components/breadcrumbs/services/link-parser.service.ts]: how console log? please pull from dev and push with "yarn lint"

File: admin-app/src/constants/colors.ts (hunk 1 above)
Comment (Solpio) [reply admin-app/src/constants/colors.ts:44]: Yes

**Pull request merged**